markhor_flippers/src/markhor_flippers.cpp

```diff
@@ -32,40 +38,184 @@ void joyCallback(const sensor_msgs::Joy::ConstPtr& joy)
     flipper_rl_pub.publish(msg);
     flipper_rr_pub.publish(msg);
   }
+  if (individual_mode == true)
+  {
+    ROS_INFO("individual mode is successful"); 
```

Comment:
Maybe add the `individual mode is successful` when changing the mode ? Correct me if I'm wrong, but doing this way, every time there's a  joyCallback and in the indivudual flippers mode, it will print this line.

Reply:
Correct, this is a relic from tests I did. My bad, i'll cull this line.